moveit_ros/planning/plan_execution/src/plan_execution.cpp

```diff
@@ -157,6 +157,31 @@ void plan_execution::PlanExecution::planAndExecute(ExecutableMotionPlan& plan,
   }
 }
 
+void plan_execution::PlanExecution::execute(ExecutableMotionPlan& plan)
+{
+  plan.planning_scene_monitor_ = planning_scene_monitor_;
+  plan.planning_scene_ = planning_scene_monitor_->getPlanningScene();
+  plan.error_code_ = executeAndMonitor(plan);
+}
+
+void plan_execution::PlanExecution::execute(ExecutableMotionPlan& plan, const moveit_msgs::PlanningScene& scene_diff)
+{
+  if (planning_scene::PlanningScene::isEmpty(scene_diff))
```

Comment:
For more consistency, please change the logic to 
if (!isEmpty(scene_diff)) {
  apply scene_diff
}
plan.error_code_ = executeAndMonitor(plan);